crates/orchestrator/src/jobs/da_job/mod.rs

```diff
@@ -272,9 +273,29 @@ async fn state_update_to_blob_data(
         blob_data.push(*compiled_class_hash);
     }
 
+    // saving the blob data of the block to S3 bucket
+    store_blob_data_s3(blob_data.clone(), block_no, config).await?;
```

Comment:
we shouldn't use `s3` anywhere outside the s3 crate, the user could be using some other storage client. we can just call it `store_blob_data`

Reply:
done